packages/compile-solidity/test/test_supplier.js

```diff
@@ -170,10 +167,23 @@ describe("CompilerSupplier", function () {
         options: cachedOptions
       });
 
-      assert(await fse.exists(expectedCache), "Should have cached compiler");
+      const cachedCompilerFilenames = fse.readdirSync(compilerCacheDirectory);
+      const compilerFilename = cachedCompilerFilenames.find(filename => {
+        return filename.includes("v0.4.21+commit.dfe3193c");
+      });
+
+      assert(
```

Comment:
Another style note -- it's better to use `assert.somethingMoreSpecific` when it exists; in this case, that could be `assert.notEqual` or `assert.notStrictEqual`.